src/ARVRInterface.cpp

```diff
@@ -39,11 +39,25 @@ void godot_arvr_set_anchor_detection_is_enabled(void *p_data, bool p_enable) {
 }
 
 godot_bool godot_arvr_is_stereo(const void *p_data) {
+	arvr_data_struct *arvr_data = (arvr_data_struct *)p_data;
 	godot_bool ret;
 
 	// TODO we should check our configuration and see if we are setup for stereo (hmd) or mono output (tablet)
-
-	ret = true;
+	if (arvr_data == nullptr || arvr_data->openxr_api == nullptr) {
+		ret = true;
+	} else {
+		switch (arvr_data->openxr_api->get_view_configuration_type()) {
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_MONO: {
+				ret = false;
+			}; break;
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO: {
+				ret = true;
+			}; break;
+			default: {
+				ret = true;
+			}; break;
```

Comment:
Whenever the cases are the same, I usually combine them as it also serves as a description to the logic (e.g: default value is `XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO`).

I think we can apply the same style here.

Reply:
Tricky, I like the explicitness of having every supported value as it's own case, that way it is clearly communicated what the intention is for that case and not that an option was forgotten and the default logic is in fact not correct.

This is also why I left out the two options we currently do not support.

Reply:
I've added placeholder code for the unsupported options for future reference, and added printing an error on the default code path to make clear an unsupported type was provided.

Reply:
Sounds good.